tests/python/relax/test_op_manipulate.py

```diff
@@ -2190,5 +2188,122 @@ def test_broadcast_to_infer_struct_info_wrong_input_type():
         bb.normalize(relax.op.broadcast_to(x1, stgt))
 
 
+def test_collapse_sum_like_infer_struct_info():
```

Comment:
We need more tests in the following aspects:
* the input tensor shape is symbolic (`test_xxx_shape_symbolic()`)
* the input tensor shape is shape var (`test_xxx_shape_var()`)
* input tensors on more dtype (`test_xxx_more_input_dtype()`)
* tests that are expected to fail in all kinds of cases